app/views/profile/show.html.erb

```diff
@@ -2,35 +2,35 @@
   <%= render 'users/header' %>
 
   <!--main content-->
-  <% unless @user.profile.title.blank? %>
-    <h3 class="heading"><%=h @user.profile.title %></h3>
+  <% if @profile.title.present? %>
+    <h3 class="heading"><%=h @profile.title %></h3>
   <% end %>
 
   <div class="wrapper">
     <dl class="meta">
       <dt class="pseuds"><%= link_to ts("My pseuds:"), user_pseuds_path(@user) %></dt>
-      <dd class="pseuds"><%= print_pseud_list(@user.pseuds) %></dd>
+      <dd class="pseuds"><%= print_pseud_list(@profile.pseuds) %></dd>
```

Comment:
It seems like there ought to be some kind of text to indicate when the user has more than just these pseuds, e.g. "and X more" or something that links to the pseuds page.

Reply:
It could be the same "All Pseuds (#)" text we use on the sidebar.